### ISSUE TITLE
allow_duplicate output fails silently

### ISSUE THREAD

**Issue by ryan-bloom** [GibbsConsulting/django-plotly-dash#481]
Re-opening this issue that was raised here as it was never addressed or solved and would be super super helpful! https://github.com/GibbsConsulting/django-plotly-dash/issues/475

TLDR: 
allow_duplicate tag on Outputs doesn't fail, but the targeted output is not updated in the DjangoDash app. The link to the issue above explains this well.

**Comment by LudoFeumo** [comment]
Please, solving this issue will be super super helpful. I am stuck on this for days.

**Comment by spiqueras** [comment]
Given that [there's no support for `allow_duplicate`](https://github.com/GibbsConsulting/django-plotly-dash/issues/475#issuecomment-1752093191), I'd be great if setting `allow_duplicate` failed hard and loudly.

**Comment by dsmi148** [comment]
> Given that [there's no support for `allow_duplicate`](https://github.com/GibbsConsulting/django-plotly-dash/issues/475#issuecomment-1752093191), I'd be great if setting `allow_duplicate` failed hard and loudly.

Agreed. Most projects I work on in Dash depend on allow_duplicate, so this is essentially a deal-breaker for me when using this package. This limitation should be prominently mentioned in the documentation, and ideally, a warning or error should be raised when allow_duplicate is used.
